app/services/audit_results.rb

```diff
@@ -39,7 +44,12 @@ class AuditResults
     UNEXPECTED_VERSION => "actual version (%{actual_version}) has unexpected relationship to %{addl} db version; ERROR!",
     INVALID_MOAB => "Invalid moab, validation errors: %{addl}",
     PC_PO_VERSION_MISMATCH => "PreservedCopy online moab version %{pc_version} does not match PreservedObject current_version %{po_version}",
-    ONLINE_MOAB_DOES_NOT_EXIST => "db has moab that is not found online"
+    ONLINE_MOAB_DOES_NOT_EXIST => "db has moab that is not found online",
+    MOAB_FILE_CHECKSUM_MISMATCH => "The checksums for %{file_path} do not match. Computed %{algorithm} checksum: %{computed_checksum[0...7]}. Expected checksum: %{expected_checksum[0...7]}",
+    FILE_NOT_IN_MOAB => "The checksums for %{manifest_file_path} refers to file (%{file}) not found in Moab",
+    FILE_NOT_IN_MANIFEST => "%{manifest_file_path} not found in Moab",
+    MANIFEST_NOT_IN_MOAB => "%{file} is missing from %{manifest_file_path}",
```

Comment:
and then should this be `"%{manifest_file_path} is missing from Moab"`?

Reply:
"%{manifest_file_path} not found in Moab"   from above error ...